tests/command_count_test.go

```diff
@@ -0,0 +1,41 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestCommandCOUNT(t *testing.T) {
+	commandsCount := 40
+
+	subscriber := getLocalConnection()
+
+	responseValue := fireCommand(subscriber, "COUNT")
+	responseCount  := responseValue.(int64)
+	if responseValue == nil {
+		t.Fail()
+	}
+
+	assert.Assert(t, responseCount == int64(commandsCount),
+		fmt.Sprintf("Unexpected number of CLI commands found. %d expected, %d found", commandsCount, responseCount))
+}
+
+func call(howmany int, t *testing.B) {
```

Comment:
Giving such generic function name is a really bad idea as this function is available throughout the package. Please reer to other benchmark functions and go's best practices and structure this better.

Also, update the output of number of allocs and time taken during the benchmar.

Reply:
Updated the method name; Followed the benchmark standards of go.
Alloc and time taken will be printed by default when the test runs.